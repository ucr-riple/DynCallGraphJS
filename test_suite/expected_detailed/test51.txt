{
    'global (test51.js:13:1:13:7)': [
        'main (test51.js:1:1:12:2)'
    ],
    'main (test51.js:10:3:10:23)': [
        'anon (test51.js:3:13:5:6)'
    ],
    'main (test51.js:11:3:11:19)': [
        'anon (test51.js:6:14:8:6)'
    ]
}