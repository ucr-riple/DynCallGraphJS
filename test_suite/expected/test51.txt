{ '(test51.js:13:1:13:7)':
   [ '(test51.js:1:1:12:2)' ],
  '(test51.js:10:3:10:23)':
   [ '(test51.js:3:13:5:6)' ],
  '(test51.js:11:3:11:19)':
   [ '(test51.js:6:14:8:6)' ] }